frodo.js

```diff
@@ -1,21 +1,21 @@
-const filteredArray = array.filter(item => item !== null && item !== undefined);
```

Comment:
SCRIPT-I like how you've used promise chaining here, it makes the code flow very clear.